### PR TITLE
add textTransformButton option to customize buttons text

### PR DESCRIPTION
**Pull request by rawle51**
# Problem
Need to customize button text
image

# Solution
add prop `textTransformButton` to `customUI` config option

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/types/ui-customisation-options.ts
Comment (it-ony) [inline src/types/ui-customisation-options.ts:43]: Possible to make it an enum, with the values from here: https://developer.mozilla.org/de/docs/Web/CSS/text-transform

**Review by stefaniacardenas — APPROVED** [review]
LGTM

File: src/types/ui-customisation-options.ts
Comment (rawle51) [reply src/types/ui-customisation-options.ts]: @it-ony yep, great suggestion.
replace type

**Review by kopijunkie — APPROVED** [review]

**Comment by rawle51** [comment]
Hi!

If everything is good, could we have the merge and update sdk version with this improvement?
What should I do to make it happens?)
 @kopijunkie @stefaniacardenas

**Comment by kopijunkie** [comment]
@rawle51 Thanks for your contribution. :) We've raised a ticket internally to have it accepted first and we'd also like to add some documentation for it before merging and releasing this improvement.

**Comment by rawle51** [comment]
To long to wait for merge....

**Pull request closed without merging**
